buildenv.mk

```diff
@@ -30,6 +30,10 @@
 #
```

Comment:
change the commit title as
"add version and build date support.

the current release version is v0.2.0.
"